pkg/transforms/common.go

```diff
@@ -40,6 +40,16 @@ func commonProperties(resource v1.Object) map[string]interface{} {
 	if resource.GetLabels() != nil {
 		ret["label"] = resource.GetLabels()
 	}
+
+	annotations := resource.GetAnnotations()
+
+	if annotations != nil {
+		// This annotation is large and useless
+		delete(annotations, "kubectl.kubernetes.io/last-applied-configuration")
+
+		ret["annotations"] = annotations
```

Comment:
Use `annotation` without the (s) to match `label`.

Reply:
Changed in the latest push.

Reply:
thanks